ui/pages/notifications/notification-components/erc721-sent-received/erc721-sent-received.tsx

```diff
@@ -112,15 +113,16 @@ export const components: NotificationComponent<ERC721Notification> = {
       type: 'body_onchain_notification',
       Image: ({ notification }) => {
         const chainId = decimalToHex(notification.chain_id);
-        const { nativeCurrencyLogo, nativeCurrencyName } =
-          getNetworkDetailsByChainId(`0x${chainId}` as keyof typeof CHAIN_IDS);
         return (
           <NotificationDetailNft
-            networkSrc={nativeCurrencyLogo}
-            tokenId={notification.data.nft.token_id}
-            tokenName={notification.data.nft.name}
-            tokenSrc={notification.data.nft.image}
-            networkName={nativeCurrencyName}
+            nft={{
+              address: notification.data.nft?.collection.address || '',
+              chainId,
```

Comment:
I've tested this, this does not work correctly.
1. We need to append this chainId with an `0x` at the start.
2. The underlying component  `nft-item.js` only uses the [current network](https://github.com/MetaMask/metamask-extension/blob/4e98d1af3c2b7aadb03335736fe663938075edf6/ui/components/multichain/nft-item/nft-item.js#L35), whereas notifications can come from any network (not just the current network selected)

Reply:
CC @MetaMask/notifications @darkwing 

If this is a PR blocker, I'm fine with ignoring for now if we can get a fix in a diff PR soon (otherwise all notifications will look like they came from the users selected chain)

Reply:
@Prithpal-Sooriya you are right

Reply:
Should be an easy fix in the `nft-item.js` component.

```js
  const { image, imageOriginal, name, tokenId, chainId } = nft;
  ...
  const currentChain = chainId ?? useSelector(getCurrentNetwork); // use the chainId if it was passed in
```